app/ts/components/pages/InterceptorAccess.tsx

```diff
@@ -1,20 +1,58 @@
 import { AddNewAddress } from './AddNewAddress.js'
-import { AddressInfoEntry, AddressBookEntry, AddingNewAddressType, RenameAddressCallBack, AddressInfo, Website, WebsiteSocket, SignerName } from '../../utils/user-interface-types.js'
-import { ExternalPopupMessage } from '../../utils/interceptor-messages.js'
+import { AddressInfoEntry, AddressBookEntry, AddingNewAddressType, RenameAddressCallBack, Website } from '../../utils/user-interface-types.js'
+import { ExternalPopupMessage, PendingAccessRequest, PendingAccessRequestArray } from '../../utils/interceptor-messages.js'
 import { sendPopupMessageToBackgroundPage } from '../../background/backgroundUtils.js'
 import Hint from '../subcomponents/Hint.js'
 import { convertNumberToCharacterRepresentationIfSmallEnough, tryFocusingTab } from '../ui-utils.js'
 import { ChangeActiveAddress } from './ChangeActiveAddress.js'
-import { DinoSays } from '../subcomponents/DinoSays.js'
+import { DinoSays, DinoSaysNotification } from '../subcomponents/DinoSays.js'
 import { getPrettySignerName } from '../subcomponents/signers.js'
 
+const HALF_HEADER_HEIGHT = 48 / 2
+
+
+function Title({ icon, title} : {icon: string | undefined, title: string}) {
+	return <p style = 'font-weight: 700; line-height: 48px'>
+		{ icon === undefined
+			? <></>
+			: <img src = { icon } style = 'width: 48px; height: 48px; vertical-align: bottom; margin-right: 10px;'/>
+		}
+		{ title }
+	</p>
+}
+
+function AccessRequestHeader(website: Website) {
+	return <header class = 'card-header' style = 'height: 40px'>
+		<div class = 'card-header-icon noselect nopointer' style = 'width: 100%;'>
+			<WebsiteOriginText { ...website } />
+		</div>
+	</header>
+}
+
+type UnderTransactionsParams = {
+	reversedPendingAccessRequestArray: PendingAccessRequestArray
+}
+
+function UnderAccesses(param: UnderTransactionsParams) {
+	const nTx = param.reversedPendingAccessRequestArray.length
+	return <div style = {`position: relative; top: ${ nTx * -HALF_HEADER_HEIGHT }px;`}>
+		{ param.reversedPendingAccessRequestArray.map((pendingAccessRequest, index) => {
+			const style = `margin-bottom: 0px; scale: ${ Math.pow(0.95, nTx - index) }; position: relative; top: ${ (nTx - index) * HALF_HEADER_HEIGHT }px;`
+			return <div class = 'card' style = { style }>
+				<AccessRequestHeader { ...pendingAccessRequest.website } />
+				<div style = 'background-color: var(--disabled-card-color); position: absolute; width: 100%; height: 100%; top: 0px'></div>
+			</div>
+		}) }
+	</div>
+}
+
 function AssociatedTogether({ associatedAddresses, renameAddressCallBack }: { associatedAddresses: readonly AddressInfoEntry[], renameAddressCallBack: RenameAddressCallBack } ) {
 	const [showLogs, setShowLogs] = useState<boolean>(associatedAddresses.length > 1)
 
 	return <>
-		<div class = 'card' style = 'margin-top: 10px; margin-bottom: 10px'>
+		<div class = 'card' style = 'margin-top: 10px; margin-bottom: 10px;'>
```

Comment:
there are `<p>` tags inside `<p>` elements within this code block which could potentially display incorrectly. consider using `<span>` to be safe

Reply:
I created issue about this. I didn't know that its not allowed